categories.php

```diff
@@ -0,0 +1,22 @@
+<?php
+header('Content-Type: application/json; charset=utf-8');
+header('Access-Control-Allow-Origin: *');
+
+require('vendor/autoload.php');
+
+use Furniture\Classes\DbConnector;
+use Furniture\Classes\CategoryHydrator;
+use Furniture\Classes\CategoryEntity;
+use Furniture\Services\ResponseService;
+
+$successMessage = "Successfully retrieved categories";
+
+try {
+    $db = DbConnector::connectToDb();
+    $categories = CategoryHydrator::fetchAllCategories($db);
+    echo json_encode(ResponseService::createResponse($successMessage, $categories));
+} catch (PDOException $exception) {
```

Comment:
what if something that isnt a PDO exception is thrown? you have no catch for this